Sources/AccessibilitySnapshot/SnapshotTesting/SnapshotTesting+Accessibility.swift

```diff
@@ -14,8 +14,14 @@
 //  limitations under the License.
 //
 
+import UIKit
 import SnapshotTesting
```

Comment:
nit: Sort the imports
```suggestion
import SnapshotTesting
import UIKit
```